Products/ZenModel/migrate/setMariaDbBufferPoolSize.py

```diff
@@ -0,0 +1,66 @@
+##############################################################################
+# 
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+# 
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+# 
+##############################################################################
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+
+sm.require("1.0.0")
+
+class SetMariaDbBufferPoolSize(Migrate.Step):
+    """Setting MariaDB buffer pool size default"""
+
+    version = Migrate.Version(5, 2, 0)
+
+    def cutover(self, dmd):
+
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        marialist = ['mariadb', 'mariadb-model']
```

Comment:
JP and Ian said that maria-events would have a different RAM requirement and they were happy that it was not included in the check.